app/controllers/api/comments_controller.rb

```diff
@@ -16,8 +15,9 @@ def create
 
   def comment_params
     params.require(:comment).permit(
+      :commentable_id,
+      :commentable_type,
```

Comment:
I noticed that before you put these in the `action` but later took them out - good that you realized that :+1: